source/PLL_API.c

```diff
@@ -0,0 +1,88 @@
+/** 
+ * @file        PLL_API.c
+ *
+ * @brief       PLL_API contains the PLL basic functions
+ *
+ * This Module implements the PLL basis functionalities. The Module contains
+ * functions to set the PLL output frequencys.
+ * 
+ * @author      Florian Heidecker
+ * @date        13.11.2015 - initial version
+ * 
+ * @version     0.1
+ *
+ */
+
+#include <xc.h>
+
+#include "log.h"
+#include "global.h"
+#include "PLL_API.h"
+
+void pll_init(){
+    LOG("PLL init\n");
```

Comment:
I think it would be better if you add a new logging level int log.h for this module.
#define PLL_LOGGING
e.g. PLL_LOG()
see. example in log.h